src/renderer/widgets/Dropdown.tsx

```diff
@@ -22,6 +22,8 @@ interface IDropdownProps {
   options: IOptionEntry[];
   value: string;
   isDisabled: boolean;
+  noSelectionValue?: JSX.Element | string;
+  noOptionValue?: JSX.Element | string;
```

Comment:
Could we rename these to be `noSelectionPlaceholder` and `noOptionPlaceholder` (or `emptyOptionsPlaceholder`)? It makes it more clear that this is something that is a placeholder and not an actual value in the dropdown imo